adsmutils/__init__.py

```diff
@@ -85,7 +87,7 @@ def get_date(timestr=None):
 
 
 
-def load_config(proj_home=None, extra_frames=0):
+def load_config(app_name, proj_home=None, extra_frames=0):
```

Comment:
if we change function signature, we'll break all repos using it; how about this?

```def load_config(proj_home=None, extra_frames=0, app_name=None):
        .....
        conf_update_from_env(app_name or conf.get('SERVICE'), conf)
```

we already use 'SERVICE' in our deployments to name the services